src/modules/user-open-orders/selectors/user-open-orders.js

```diff
@@ -11,17 +11,32 @@ import { CLOSE_DIALOG_CLOSING } from 'modules/market/constants/close-dialog-stat
 
 import { formatNone, formatEtherTokens, formatShares } from 'utils/format-number'
 
+export function selectAllUserOpenOrderMarkets() {
+  const { loginAccount, orderBooks } = store.getState()
+
+  if (!loginAccount.address || orderBooks == null) {
+    return []
+  }
+
+  return Object.keys(orderBooks)
+    .filter(marketID => Object.keys(orderBooks[marketID])
+      .filter(outcome => Object.keys(orderBooks[marketID][outcome])
+        .filter(type => Object.keys(orderBooks[marketID][outcome][type])
+          .filter(hash => orderBooks[marketID][outcome][type][hash].owner === loginAccount.address))))
+}
+
 /**
- *
+ * Pulls off existing order book in state
  * @param {String} outcomeId
- * @param {{buy: object, sell: object}} marketOrderBook
+ * @param {String} marketID
  *
  * @return {Array}
  */
-export default function (outcomeId, marketOrderBook) {
```

Comment:
I’m not a fan of this change — let’s go ahead and roll this all back.  

We should be trying to diagnose the core issue of why it wasn’t picking up the state change prior to refactoring other methods.

Reply:
reverted user-open-orders to orig. 
modified markets-all to use market.js selectMarket instead of assembleMarket